nextjs-13/prisma/schema.prisma

```diff
@@ -8,13 +8,28 @@ datasource db {
 }
 
 model User {
-  id         String   @id @default(uuid())
-  name       String   @db.VarChar(100)
-  email      String   @db.VarChar(50)
-  avatarUrl  String?  @map("avatar_url")
-  createdAt  DateTime @map("created_at") @default(now())
+  id         String       @id @default(uuid())
+  name       String       @db.VarChar(100)
+  email      String       @db.VarChar(50)
+  avatarUrl  String?      @map("avatar_url")
+  createdAt  DateTime     @default(now()) @map("created_at")
+  updatedAt  DateTime     @default(now()) @map("updated_at")
+  Newsletter Newsletter[]
 
-  @@map("users")
   @@unique([email])
   @@index([email])
+  @@map("users")
+}
+
+model Newsletter {
+  id        String   @id @default(uuid())
+  name      String   @db.VarChar(255)
+  content   String   @db.Text()
+  user      User     @relation(fields: [userId], references: [id])
+  userId    String   @map("user_id")
+  createdAt DateTime @default(now()) @map("created_at")
+  updatedAt DateTime @default(now()) @map("updated_at")
```

Comment:
For the `@updateAt` I think you can the field type to @updateAt this auto update when the row got an update. [Reference](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#updatedat)

Reply:
Ah right, I forgot that. Good catch, thank you! 🚀 

The fix: https://github.com/nimblehq/js-framework-benchmark/pull/96/commits/9f9d399a0e891cb765de1e5166e04419518888c7

Reply:
Maybe on the user model too

Reply:
Oh you already added, Sorry I couldn't see in the fix file 😅